doc/usage.rst

```diff
@@ -480,3 +480,56 @@ GitHub
 - *MERGE_REQUEST_ID* is the pull request number.
 
 .. _`personal access token`: https://github.blog/2013-05-16-personal-api-tokens/
+
+.. _traceability_jira_automation:
+
+--------------------
+Jira ticket creation
+--------------------
+
+Traceable items can be configured to create Jira tickets for. Duplication of tickets is avoided by querying Jira
+first for existing tickets based on the Jira project and ticket summary. Below is an example configuration:
+
+Configuration
+=============
+
+.. code-block:: python
+
+    traceability_jira_automation = {
+        'api_endpoint': 'https://jira.atlassian.com/rest/api/latest/',
+        'username': 'my_username',
+        'password': 'my_password',
+        'project_key_regexp': r'ACTION-(?P<project>\d{5})_',
+        'project_key_prefix': 'MLX',
+        'default_project': 'SWCC',
+        'issue_type': 'Task',
+        'item_to_issue_regex': r'ACTION-12345_ACTION_\d+',
```

Comment:
Would a key `'item_to_ticket_regex'` be a better fit?

Reply:
Yes, since jira jargon is ticket

Reply:
Great, done in cc3aa7da57e3a40c146fd731df0dba4ffb82dbcf